### PR TITLE
Added Labels Feature

### PR DESCRIPTION
**Pull request by nikhil-sharma-nks**
Changelogs
- Added Labels feature
- Added new color codes for notes background
labels

https://04-labels--noting-note-app.netlify.app/home

### CODE REVIEW

File: src/components/Notes/NoteCardView/NoteCardView.jsx [hunk 1]
```diff
@@ -63,6 +65,37 @@ const NoteCardView = ({ note, fromArchive }) => {
     }
   };
 
+  const checkIfChecked = (label) => {
+    return tags.includes(label);
```
Comment (atul-gairola) [inline src/components/Notes/NoteCardView/NoteCardView.jsx:86]: nothing wrong here, but this function could've just been an individual inline function. Good use of includes method here.

**Review by atul-gairola — APPROVED** [review]
Looks good to merge

File: src/components/Notes/NoteCardView/NoteCardView.jsx [hunk 2]
```diff
@@ -63,6 +65,37 @@ const NoteCardView = ({ note, fromArchive }) => {
...
+    if (isChecked) {
+      setUpdatedNoteTags((s) => {
+        handleUpdate('TAG', [...s, label]);
+        return [...s, label];
+      });
+    } else {
+      const newSet = new Set(updatedNoteTags);
+      newSet.delete(label);
+      setUpdatedNoteTags([...newSet]);
+      handleUpdate('TAG', [...newSet]);
```
Comment (HarshnaNHaswani) [inline src/components/Notes/NoteCardView/NoteCardView.jsx:98]: nice use of set

**Review by HarshnaNHaswani — COMMENTED** [review]
Good to go
- neat folder structure
- clean, formatted code
- good data flow
- suggestion: can use semantic html more - lists or sections

**Pull request merged**